react-context/my-app/src/components/useCart.ts

```
@@ -0,0 +1,6 @@
+import { useContext } from 'react';
+import { CartContext } from './CartContent';
+
+export function useCart() {
+  return useContext(CartContext);
+}
```

There's a chance that `useContext(CartContext)` can returned undefined if a component that's not wrapped in a provider tries to access this. So rather than returning it directly, it'd be better to assign the result of `useContext` to a variable, and check if that variable is truthy so we can throw an error if not